src/cace/ari/text_util.h

```diff
@@ -140,6 +140,22 @@ int slash_unescape(string_t out, const cace_data_t *in);
  */
 void strip_space(string_t out, const char *in, size_t in_len);
 
+/** Convert a text string to lowercase.
+ * This is written to work on byte strings, not unicode.
+ *
+ * @param[out] out The output buffer, which will be replaced.
+ * @return Zero upon success.
+ */
+int cace_string_tolower(string_t out);
```

Comment:
If it really cannot fail then these can return void and use precond macro `CHKVOID()`. Since the return values were not being checked anyway.

Reply:
Thanks @BrianSipos, changed both functions to `void`.